store/tikv/snapshot.go

```diff
@@ -330,15 +335,10 @@ func extractKeyErr(keyErr *pb.KeyError) error {
 
 func newWriteConflictError(conflict *pb.WriteConflict) error {
 	var buf bytes.Buffer
-	_, err := fmt.Fprintf(&buf, "txnStartTS=%d, conflictTS=%d, key=",
-		conflict.StartTs, conflict.ConflictTs)
-	if err != nil {
-		logutil.Logger(context.Background()).Error("error", zap.Error(err))
-	}
 	prettyWriteKey(&buf, conflict.Key)
 	buf.WriteString(" primary=")
 	prettyWriteKey(&buf, conflict.Primary)
-	return ErrWriteConflict.FastGen(buf.String())
```

Comment:
Maybe we should enhance FastGen by use https://github.com/pingcap/errors/pull/13 :D

Reply:
for high conflict situation, control logic error should never generate stack.